guide/src/class.md

```diff
@@ -891,6 +890,52 @@ impl PyIterProtocol for Container {
 For more details on Python's iteration protocols, check out [the "Iterator Types" section of the library
 documentation](https://docs.python.org/3/library/stdtypes.html#iterator-types).
 
+#### Returning a value from iteration
```

Comment:
I think `PyIterProtocols`'s doc would be a better place for this document. `class.md` is too long now.